docs/slotfilling.rst

```diff
@@ -192,3 +192,254 @@ like ``MyCustomFormField`` and overriding the ``validate()`` method.
    Typically your NLU model will assign this free-text input to 2-3 different intents. 
    It's easiest to add stories for each of these. 
 
+Slot Filling with Forms
+-----------------------
+
+Another alternative is to use Forms, an object which supercedes the policy ensemble and carries out deterministic logic for the purpose of filling slots.
+
+This is an extension on the ``FormAction`` for a number of reasons:
+
+- It no longer requires a change in stories if the structure of the form is changed. I.e. if an extra slot is required to be filled then the flow can be changed simply by editing an object instead of rewriting stories.
+
+- The intent is ignored except for in specific cases where the user does not want to fill out the form. (for example if they say goodbye).
+
+- There can be conditional logic on key-value pairs. So if a certain slot is set by the questioning, the remaining slots to be filled can be altered.
```

Comment:
maybe mention that this feature is in beta? bc not really tested with anyone yet right?